_drafts/newsletter-001.md

```diff
@@ -42,6 +42,11 @@ sharing that information is optional.
 Only 3 questions in this survey are mandatory and we've saved
 the most important one for last.
 
+The survey is soft-closed now
+as existing responses are being processed for publishing,
+but we’re still accepting responses until the survey
+has been publishes & subsequently closed for good.
```

Comment:
I'm not quite sure what this means, "closed, but not really"? Also, since this is the first newsletter, maybe just not mention this, and let people respond, who knows how many more responses will come in from the exposure of this newsletter.

Reply:
@erlend-sh ping

Reply:
Let’s just cut out the soft-closed part:

“The survey is now being processed for publishing, but we’re still accepting responses until the survey has been published and subsequently closed for good.”

The reach of the newsletter isn’t going to be meaningfully different from our announcement post, so we might as well mention it again here; I’m not concerned it will get too much exposure.

Reply:
Done!